source/tutorials/filtering.rst

```diff
@@ -156,7 +162,74 @@ This syntax also supports negating individual clauses of the ``--p-where`` expre
      --p-where "Subject='subject-1' AND NOT BodySite='gut'" \
      --o-filtered-table subject-1-non-gut-filtered-table.qza
 
-.. note:: Currently, the most common metadata-based filtering of features is based on feature taxonomy, such as filtering all features that are annotated as being in a particular genus. This can currently be achieved using ``filter-features`` if taxonomy is provided in a feature metadata file. We are working on adding more direct support for this functionality, which will be made available in a new method of the ``q2-taxa`` plugin. You can track progress on this `here <https://github.com/qiime2/q2-taxa/issues/40>`_.
+Taxonomy-based filtering of tables and sequences
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Taxonomy-based filtering is a very common type of feature-metadata-based filtering, so the ``q2-taxa`` plugin provides the ``filter-table`` method to simplify this process. Filtering can be applied to retain only specific taxa using ``--p-include`` and/or to remove specific taxa using ``--p-exclude``.
+
+Removing a feature if its taxonomic annotation contains some specific text is achieved with the ``--p-exclude`` parameter. For example, ``--p-exclude`` is used here to remove all features annotated as ``mitochondria`` from a table. When searching with ``--p-mode contains`` (the default), search terms are case insensitive, so the search term ``mitochondria`` would return the same results as the search term ``Mitochondria``.
+
+.. command-block::
+   qiime taxa filter-table \
+     --i-table table.qza \
+     --i-taxonomy taxonomy.qza \
+     --p-exclude mitochondria \
+     --o-filtered-table table-no-mitochondria.qza
+
+Removing features that match more than one search term is achieved by providing the search terms in a comma-separated list. The following command will remove all features that contain either ``mitochondria`` or ``chloroplast`` in their taxonomic annotation.
+
+.. command-block::
+   qiime taxa filter-table \
+     --i-table table.qza \
+     --i-taxonomy taxonomy.qza \
+     --p-exclude mitochondria,chloroplast \
+     --o-filtered-table table-no-mitochondria-no-chloroplast.qza
+
+Filtering a table to retain only specific features is achieved using the ``--p-include`` parameter. For example, ``--p-include`` can be used to retain only features that were at least annotated to the phylum level. In the Greengenes taxonomy (which was used to generate the ``FeatureTable[Taxonomy]`` being provided here), all phylum annotations begin with the text ``p__``. We can therefore use ``p__`` as a ``--p-include`` include term here to retain only features that contain any phylum-level annotation. In practice, this filter might be useful for filtering features that are unlikely to be taxonomically informative about your samples.
```

Comment:
I was kind of hoping the last sentence would explain that not all features may have been classified up to a phylum level. Unless you realize that, it's easy to misunderstand why ``p__`` is helpful. What's written hints at that, but it doesn't explain why the filter works.